data/coalition/coalition missions.txt

```diff
@@ -3167,3 +3166,450 @@ mission "Heliarchs Buy Jump Drive Mission"
+			choice
+				`	"They have a translation device."`
+				`	"I'm sure they can learn the language."`
+			`	"That's reassuring... but, how am I even going to get there?"`
+			choice
+				`	"I'll take you there."`
+			`	"Do I have to pay anything? As a struggling author, I can't afford off-world transport at all, let alone to a completely different alien race!"`
+			choice
+				`	"It's okay. I'm willing to make an exception for you."`
+			`	"Thank you so much! Just wait for a while, I need to finish my book signing event. It should be roughly 45 minutes before it's over, and then we can go to whatever this Coalition thing is," says Anna.`
+			`	You reply, "Good luck."`
+			`	As you wait for Anna, you look over your maps of Coalition space to determine the most suitable place for Anna. After much deliberation and thought, you decide on <destination>.`
+			`	After approximately an hour of waiting, Anna appears outside of your ship, knocking on the hatch. You open the hatch to let her inside, as you lead her to her bunk and inform her of the destination. You prepare to leave the spaceport.`
+				accept
+	on complete
+		conversation
+			`	You leave the ship with Anna following you. The moment you are out of your ship, three Kimek surround you and Anna. The Kimek in the middle speaks, and then brief silence. The silence is broken as their translation box speaks.`
+			`	"Greetings, <first> <last>. This other human you have brought, who she is, may I ask you?"`
+			`	You and Anna introduce yourself. Anna's introduction is longer, while you only need to state your name, seeing as you are more familiar with the Coalition now.`
+			`	"About what, do you write?", asks the Kimek.`
+			`	"Oh, I write science fiction books," says Anna.`
+			`	The Kimek begin conversing amongst themselves in their native language for a good ten to twenty seconds before they move to face you again.`
+			`	Then, switching back to your language, the Kimek explains that Anna is welcome to be with you, but states that she will not be allowed to do much unless she registers.`
+			`	"Register?", asks Anna.`
+			`	"You have to register yourself with the Coalition before you can do most stuff here. I'll take you to the Ring of Friendship, that's where you can complete the registration process."`
+			`	"Okay," says Anna, "I'll explore the spaceport a little bit to find some inspiration. Get your ship ready and meet me there."`
+
+mission "Fangvin: Struggling Author 2"
+	name "Help Anna Register"
+	description "Take Anna Fangvin to the <destination> so she can register with the Heliarchs."
+	source "Ki Patek Ka"
+	destination "Ring of Friendship"
+	to offer
+		has "Fangvin: Struggling Author 1: done"
+	on offer
+		conversation
+			`You walk through the spaceport, looking for Anna. She is at a restaurant, but has ordered nothing with a disgusted look on her face. "These aliens! They vomit on their food and slurp all the soup! Disgusting!", she mutters.`
```

Comment:
Probably shouldn't be saying this, even in muttering, when she knows these aliens have translators. Even if none she can see has a translator at hand, it's still unwise to be speaking like this about the aliens you just met, when they could potentially understand you with just one passerby having a translator box.